x-pack/solutions/observability/plugins/observability_agent/server/tools/get_change_point/index.ts

```diff
@@ -0,0 +1,309 @@
+                  Do not regurgitate these results back to the user.
+                  Instead, focus on the interesting changes, mention possible correlations or root causes, and suggest next steps to the user.
+                  "indeterminate" means that the system could not detect any changes.`,
+                  changes: {
+                    logs: allLogChangesWithoutTimeseries,
+                  },
+                },
+                data: {
+                  changes: {
+                    logs: allLogChanges,
+                  },
+                },
+              },
+            },
+          ],
+        };
+      } catch (error) {
+        logger.error(`Error getting log changes: ${error.message}`);
+        logger.debug(error);
+        return {
+          results: [
+            {
+              type: ToolResultType.error,
+              data: {
+                message: `Error getting log changes: ${error.message}`,
+                stack: error.stack,
+              },
+            },
+          ],
+        };
+      }
+    },
+  };
+
+  return toolDefinition;
+}
+
+const getMetricChangesSchema = z.object({
+  start: z
+    .string()
+    .describe('The beginning of the time range, in datemath, like now-24h, or an ISO timestamp'),
+  end: z.string().describe('The end of the time range, in datemath, like now, or an ISO timestamp'),
+  metrics: z
+    .array(
+      z.object({
+        name: z.string().describe('The name of this set of metrics'),
+        index: z.string().describe('The index or index pattern where to find the metrics'),
+        kqlFilter: z
+          .string()
+          .describe('A KQL filter to filter the log documents by, e.g. my_field:foo')
+          .optional(),
+        field: z
+          .string()
+          .describe(
+            'Metric field that contains the metric. Only use if the metric aggregation type is not count.'
+          )
+          .optional(),
+        type: z
+          .enum(['count', 'avg', 'sum', 'min', 'max', 'p95', 'p99'])
+          .describe('The type of metric aggregation to perform. Defaults to count')
+          .optional(),
+        groupBy: z
+          .array(z.string())
+          .describe('Optional keyword fields to group metrics by.')
+          .optional(),
+      })
+    )
+    .describe(
+      'Analyze changes in metrics. DO NOT UNDER ANY CIRCUMSTANCES use date or metric fields for groupBy, leave empty unless needed.'
+    ),
+});
+
+export async function createObservabilityGetMetricChangePointTool({
+  core,
+  plugins,
+  logger,
+}: {
+  core: CoreSetup<ObservabilityAgentPluginStartDependencies, ObservabilityAgentPluginStart>;
+  plugins: ObservabilityAgentPluginSetupDependencies;
+  logger: Logger;
+}) {
+  const toolDefinition: BuiltinToolDefinition<typeof getMetricChangesSchema> = {
+    id: OBSERVABILITY_GET_METRIC_CHANGE_POINT_TOOL_ID,
+    type: ToolType.builtin,
+    description: 'Returns change points like spikes and dips for metrics.',
```

Comment:
To be consistent with other tool descriptions:
```suggestion
    description: 'Return change points like spikes and dips for metrics.',
```